test/contrib/test_securetransport.py

```diff
@@ -10,6 +10,8 @@
 except ImportError:
     pass
 
+pytestmark = pytest.mark.skip()
```

Comment:
?

Reply:
No. :) It doesn't work on macOS 10.15, so I use to run tests locally, and I forgot to remove it. Thanks!